### ISSUE TITLE
`apugli:edible_item` consumes 2 items on some weird conditions

### ISSUE THREAD

**Issue by CyraLX** [MerchantPug/apugli#12]
As title suggests, with the following power the amount of items consumed is 2 for some reason:
```json
{
        "type": "apugli:edible_item",
        "item_condition": {
            "type": "apoli:food"
        },
        "food_component": {
            "hunger": 0,
            "saturation": 0
        },
        "use_action": "eat"
}
```

**Comment by ChrysanthCow** [comment]
Bug confirmed, trying to fix it now.

**Comment by ChrysanthCow** [comment]
Fixed on my end. I'm unsure how the new changes will co-operate with the Modify Food power but you can report any other bugs to me.

Will be released in Apugli v1.8.0
